### DISCUSSION TITLE
Why is this happening?

### DISCUSSION THREAD

**Discussion by SwiftSpeed01** [black7375/Firefox-UI-Fix#417] [Q&A] [answered]
The tool bar is dark and the rest is firefox dark?

![image](https://user-images.githubusercontent.com/105238672/174313782-b0933869-b879-406f-a9fd-dc0451856130.png)

**Comment by black7375** [comment, marked as answer]
Because of [`userChrome.theme.system_default`](https://github.com/black7375/Firefox-UI-Fix/blob/a5d6ad9fcbc289b1cfcf0c1e373da44cfb887e34/user.js#L125) option.
It provides a theme close to each OS look and feel.
In the case of Windows10, it has been transplanted from [Legacy Edge](https://en.wikipedia.org/wiki/Microsoft_Edge#Edge_Legacy_(2014%E2%80%932019)).

If you don't like it, modify it to `false` in `user.js` file or activate a dark theme.
